.ci/packer_cache_for_branch.sh

```diff
@@ -49,7 +49,8 @@ tar -cf "$HOME/.kibana/bootstrap_cache/$branch.tar" \
   .chromium \
   .es \
   .chromedriver \
-  .geckodriver;
+  .geckodriver \
+  .yarn-local-mirror;
```

Comment:
Any idea how long it takes to add the yarn-local-mirror to the packer cache? It seems unnecessary since the files in that directory can be shared amongst all Kibana checkouts on a machine safely. It feels like it would be better to put the mirror in the user's home directory or something so that all checkouts could share it, and so it didn't have to be extracted before every CI run.

Reply:
I assume it won't take too long as the content of that folder are `tars` too. Do you know how can we make a test to see how much time it adds? I was trying to not put it in the user directory in order to avoid to write things into the `$HOME` folder in the development environment or to avoid passing an extra flag to setup this on the CI. Do you have any other thoughts on that one?

Reply:
I think the best test I can think of is just to try it out locally. Try adding the yarn-local-mirror to a tarball on your machine, how does that time compare the the time savings of using the offline mirror? I don't see a problem with writing into the home folder on development machines, it's the closest thing we have to a known location that the user has permission to write to, and the items written there are immutable and versioned so they can be shared across many repos and installs if the user chooses.

Reply:
@spalger I've run `time tar -cf yarn-local-mirror.tar.gz .yarn-local-mirror` which is the same thing we're doing on CI. That is the time it logged:

```
real	0m2.067s
user	0m0.118s
sys	0m0.719s
```
It seems pretty fast